src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isClassTabSelected = cn({
+              'is-active': selectedTab.id === tab.id,
+            });
```

Comment:
- Why?) Just store `const isTabSelected = selectedTab.id === tab.id;` as I recomended.
- If you store classes names should be different, as it's not a boolean